es-app/src/guis/GuiMetaDataEd.cpp

```diff
@@ -50,16 +52,20 @@ GuiMetaDataEd::GuiMetaDataEd(Window* window, MetaDataList* md, const std::vector
 	// populate list
 	for(auto iter = mdd.cbegin(); iter != mdd.cend(); iter++)
 	{
-		std::shared_ptr<GuiComponent> ed;
-
 		// don't add statistics
 		if(iter->isStatistic)
 			continue;
 
+		std::shared_ptr<GuiComponent> ed;
+
 		// create ed and add it (and any related components) to mMenu
 		// ed's value will be set below
 		ComponentListRow row;
-		auto lbl = std::make_shared<TextComponent>(mWindow, Utils::String::toUpper(iter->displayName), Font::get(FONT_SIZE_SMALL), 0x777777FF);
+		auto lblTxt = Utils::String::toUpper(iter->displayName);
```

Comment:
Add a hint about the date format, as the MM/DD/YY can be confusing if the day-of-month is less than 12.